docs/settings/fleet-settings.asciidoc

```diff
@@ -167,40 +203,66 @@ NOTE: The `xpack.fleet.outputs` settings are intended for advanced configuration
   `is_default_monitoring`::: 
     If `true`, the output specified in `xpack.fleet.outputs` will be the one used to send agent monitoring data unless there is another one configured specifically for the agent policy.
 =====
+
+`xpack.fleet.fleetServerHosts`::
+List of {fleet-server} hosts that are configured when the {fleet} app starts.
 +
-Example configuration:
+.Required properties of `xpack.fleet.fleetServerHosts`
+[%collapsible%open]
+=====
+  `id`:::
+    Unique ID for the host server.
+  `name`::: 
+    Name of the host server.
+  `host_urls`::: 
+    Array of one or more host URLs that {agents} will use to connect to {fleet-server}.
+=====
 +
-[source,yaml]
-----
-xpack.fleet.packages:
-  - name: apache
-    version: 0.5.0
+.Optional properties of `xpack.fleet.fleetServerHosts`
+[%collapsible%open]
+=====
+  `is_default`:::
+    Whether or not this host should be the default to use for {fleet-server}.
+  `proxy_id`::: 
+    Unique ID of the proxy to access the {fleet-server} host.
+=====
 
-xpack.fleet.agentPolicies:
-  - name: Preconfigured Policy
-    id: 1
-    namespace: test
-    package_policies:
-      - package:
-          name: system
-        name: System Integration
-        id: preconfigured-system
-        inputs:
-          - type: system/metrics
-            enabled: true
-            vars:
-              - name: system.hostfs
-                value: home/test
-            streams:
-              - data_stream:
-                  dataset: system.core
-                enabled: true
-                vars:
-                  - name: period
-                    value: 20s
-          - type: winlog
-            enabled: false
-----
+`xpack.fleet.proxy`::
+List of proxies to access {fleet-server} that are configured when the {fleet} app starts.
++
+.Required properties of `xpack.fleet.proxy`
+[%collapsible%open]
+=====
+  `id`:::
+    Unique ID of the proxy to access the {fleet-server} host.
+  `name`::: 
+    Name of the proxy to access the {fleet-server} host.
+  `url`::: 
+    URL that {agents} use to connect to the proxy to access {fleet-server}.
+=====
++
+.Optional properties of `xpack.fleet.proxy`
+[%collapsible%open]
+=====
+  `proxy_headers`:::
+    Map of headers to use with the proxy.
+.Properties of `proxy_headers`
+[%collapsible%open]
+=======
+  `key`:::: 
+    Key to use for the proxy header.
+  `value`:::: 
+    Value to use for the proxy header.
+=======
+  `certificate_authorities`::: 
+    Certificate authority (CA) used to issue the certificate.
+  `certificate`::: 
+    The name of the certificate to use to validate the proxy.
+  `certificate_key`::: 
+    Hash pin used for certificate verification. The pin is a base64-encoded string of the SHA-256 fingerprint.
```

Comment:
I'm not sure about this one. I copied it from `xpack.fleet.agents.elasticsearch.ca_sha256`.

Reply:
@kilfoyle `certificate` and `certificate_key` are usefull to do mutual tls and are used to authenticate to the proxy

Reply:
Sounds good! I've changed the definitions to:
```
  `certificate`::: 
    The name of the certificate used to authenticate the proxy.
  `certificate_key`::: 
    The certificate key used to authenticate the proxy.
```
But let me know if you think that's not clear enough.